Source/.clang-format

```diff
@@ -4,19 +4,19 @@ AccessModifierOffset: -2
 AlignAfterOpenBracket: Align
 AlignConsecutiveAssignments: false
 AlignConsecutiveDeclarations: false
-AlignEscapedNewlinesLeft: false
+AlignEscapedNewlines: DontAlign
```

Comment:
It seems `AlignEscapedNewlinesLeft` was long-ago replaced with `AlignEscapedNewlines`.
The default value appears to be `Right`, which is how our files are currently formatted.